packages/debug/src/browser/debug-configuration-model.ts

```diff
@@ -75,9 +80,20 @@ export class DebugConfigurationModel implements Disposable {
                 }
             }
         }
+        const compounds: DebugCompound[] = [];
+        if (value && typeof value === 'object' && 'compounds' in value) {
+            if (Array.isArray(value.compounds)) {
+                for (const compound of value.compounds) {
+                    if (DebugCompound.is(compound)) {
+                        compounds.push(compound);
+                    }
+                }
+            }
+        }
```

Comment:
Minor: This can be simplified to
```suggestion
        if (value && typeof value === 'object' && Array.isArray(value.compounds)) {
            for (const compound of value.compounds) {
                if (DebugCompound.is(compound)) {
                    compounds.push(compound);
                }
            }
        }
```

Reply:
Agreed, I blindly copied the code for the configurations handling. I updated both loops with the nicer code.